lib/Doctrine/ORM/Mapping/ClassMetadataFactory.php

```diff
@@ -753,4 +752,13 @@ protected function isEntity(ClassMetadataInterface $class)
     {
         return isset($class->isMappedSuperclass) && $class->isMappedSuperclass === false;
     }
+
+    private function getTargetPlatform()
```

Comment:
Can you add a simple DocBlock here with return type, please? Would help the IDE I suppose...